runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/ProcessBuildView.tsx

```diff
@@ -59,7 +63,13 @@ const ProcessBuildView: FC = () => {
     useEffect(() => {
         if (!hasAdvancedActionsAccess) return;
         hasActionsAccess && dispatch(getActions());
-        dispatch(globalVariableActions.getGlobalVariables());
+
+        const userGlobalVariables = isAdmin ? {} : {
+            filter: {
+                equals: { 'creatorId': user.id },
+            }
+        };
+        dispatch(globalVariableActions.getGlobalVariables(userGlobalVariables));
```

Comment:
should not depend on query param. Requester ID (the one in JWT) should be taken into account